File: be/airbnb/src/main/java/com/team19/airbnb/entity/Location.java
package com.team19.airbnb.entity;

public class Location {

    private String address;
    private Double latitude;
    private Double longitude;

    Location(String address, Double latitude, Double longitude) {
        this.address = address;
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public Location create(String address, Double latitude, Double longitude) {
        return new Location(address, latitude, longitude);
    }
}

File: iOS/airbnb/airbnb/AccommodationConditions/CommonModels/Location.swift
//
//  LocationSearchResult.swift
//  airbnb
//
//  Created by Song on 2021/05/20.
//

import Foundation

struct Location: Decodable {
    let name: String
    let coordinate: Coordinate
}

struct Coordinate: Decodable {
    let latitude: Double
    let longitude: Double
}
